### PR TITLE
doc: simplify ockam enroll in quick start

### PR DESCRIPTION
**Pull request by nazmulidris**

### CODE REVIEW

**Comment by nazmulidris** [comment]
@CAOakleyII - Based on our convo yesterday, I've updated the README to simplify the commands needed to run ockam enroll & cleaned up the names of the nodes as well.

**Comment by nazmulidris** [comment]
@mrinalwadhwa I was able to add some drawings to this branch that I've been working on. Unfortunately quite a few other files were changed as well, and I'm not really sure why.

We just just discard this branch if needed.

**Comment by mrinalwadhwa** [comment]
We want to change this overall example to, let's discuss this in the next few days.

```
curl --proto '=https' --tlsv1.2 -sSfL https://install.command.ockam.io | bash
ockam enroll

python3 -m http.server --bind 127.0.0.1 5000
ockam tcp-outlet create --to 5000
ockam relay create

ockam tcp-inlet create --from 6000
curl --head 127.0.0.1:7000
```

**Comment by mrinalwadhwa** [comment]
> Should ockam tcp-inlet create --from *6000* be ockam tcp-inlet create --from *7000*?
yes :)

**Comment by nazmulidris** [comment]
> > Should ockam tcp-inlet create --from _6000_ be ockam tcp-inlet create --from _7000_?
> 
> yes :)

👍🏽 😄  

I took a first pass the narrative & example update, WDYT?
https://github.com/build-trust/ockam-documentation/pull/37/commits/5d40064def3eb7447f0a44bdb65881f132013422

**Comment by nazmulidris** [comment]
Should we also update the following article [Relays and Portals/Portals](https://docs.ockam.io/reference/command/advanced-routing#portal) with these new simplified portal commands?

It would replace this snippet that's on that article currently:
```
» python3 -m http.server --bind 127.0.0.1 9000

» ockam tcp-outlet create --at n3 --from /service/outlet --to 127.0.0.1:9000
» ockam tcp-inlet create --at n1 --from 127.0.0.1:6000 \
    --to /worker/603b62d245c9119d584ba3d874eb8108/service/forward_to_n3/service/hop/service/outlet

» curl --head 127.0.0.1:6000
HTTP/1.0 200 OK
...
```

**Comment by nazmulidris** [comment]
@mrinalwadhwa 
Should we also update [Secure Channels/Elastic Encrypted Relays](https://docs.ockam.io/reference/command/secure-channels#elastic-encrypted-relays) w/ these new streamlined commands? Drop the `--output json > project.json` and `--project-path project.json`?

```
» ockam project information --output json > project.json

» ockam node create a --project-path project.json
» ockam node create b --project-path project.json

» ockam relay create b --at /project/default --to /node/a
     ✔︎ Now relaying messages from /project/default/service/70c63af6590869c9bf9aa5cad45d1539 → /node/a/service/forward_to_b

» ockam secure-channel create --from a --to /project/default/service/forward_to_b/service/api \
    | ockam message send hello --from a --to -/service/uppercase
HELLO
```

**Comment by nazmulidris** [comment]
@etorreborre @mrinalwadhwa - Eric and I met today and he made a great suggestion for the quick start guide - the idea is to take the diagram from basic web app ex and make a version of it that is added just before we show users the quick start steps:
here 


The thinking behind it is to show the "why" first, then the "what", and finally the "how". 
- The image would convey to the user why Ockam is important and what the commands they are about to type will actually do. 
- Which is to end to end encrypt the connection between curl and the python server.

Thoughts?

**Review by etorreborre — APPROVED** [review]
I indeed think that a picture would be great on the `README` since this is the first page that people see on https://docs.ockam.io. But this can be left to another PR

File: README.md [hunk 1]
```diff
@@ -58,30 +58,29 @@ Next, step through the following commands to setup secure and private communicat
...
 
 # -- APPLICATION CLIENT --
 
-# Setup an ockam node, called `c`, as a sidecar next to our application client. Then create
-# an end-to-end encrypted secure channel with s, through the cloud relay. Finally, tunnel
-# traffic from a local tcp inlet through this end-to-end secure channel.
-ockam node create c --project-path default-project.json
-ockam secure-channel create --from c --to /project/default/service/forward_to_s/service/api\
-  | ockam tcp-inlet create --at c --from 127.0.0.1:7000 --to -/service/outlet
+# Setup a a local tcp inlet to allow raw TCP traffic to be received on port `7000` before
```
Comment (etorreborre) [inline README.md:80]: I guess it should either be `tcp-inlet` or `TCP inlet` here

File: README.md [hunk 2]
```diff
@@ -58,30 +58,29 @@ Next, step through the following commands to setup secure and private communicat
...
-ockam project information --output json > default-project.json
 
 # -- APPLICATION SERVICE --
 
-# Start an application service, listening on a local ip and port, that clients would access
-# through the cloud encrypted relay. We'll use a simple http server for this first example
+# Start an application service, listening on a local IP and port, that clients would access
+# through the cloud encrypted relay. We'll use a simple HTTP server for this first example
 # but this could be any other application service.
 python3 -m http.server --bind 127.0.0.1 5000
```
Comment (mrinalwadhwa) [inline README.md:67]: Let's change the port here to 6000, we know that 5000 conflicts on macos sometimes.

**Comment by nazmulidris** [comment]
TODO: Apply the changes in the Quickstart example [here](https://github.com/build-trust/ockam-documentation/pull/37/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) to the README to the /ockam repo.

**Comment by nazmulidris** [comment]
> I indeed think that a picture would be great on the `README` since this is the first page that people see on https://docs.ockam.io. But this can be left to another PR

@etorreborre 
I am meeting w/ Mrinal next wk to discuss images in general for the website. We can make another PR to add an image to this quick start guide after that 👍🏽

File: README.md (hunk 2 above)
Comment (nazmulidris) [reply README.md:67]: Done 👍

File: README.md (hunk 1 above)
Comment (nazmulidris) [reply README.md:80]: Done 👍

**Pull request merged**